prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+/* your styles here 👇 */
+
+div {
+  flex-basis: calc(100% / 6);
```

Comment:
Instead of targeting all divs, try targeting the `.card-container` elements only.
https://www.w3schools.com/css/css_selectors.asp